src/payments/daily-refill/notify.ts

```
@@ -0,0 +1,142 @@
+import logger from "@/utils/logger";
+import { createApnsService } from "@/api/v2/notifications/apns-push.service";
```

ESLint

[error] Delete `import·logger·from·"`@/utils/logger`";⏎`

(prettier/prettier)